scipy/stats/_hypotests.py

```diff
@@ -141,46 +141,59 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     return Epps_Singleton_2sampResult(w, p)
 
 
-PoissonMeansTestResult = namedtuple('PoissonMeansTestResult', ('statistic', 'pvalue'))
+PoissonMeansTestResult = make_dataclass('PoissonMeansTestResult',
+                                        ('statistic', 'pvalue'))
 
 
-def poisson_means_test(count1, nobs1, count2, nobs2, diff=0, alternative='two-sided'):
+def poisson_means_test(count1, nobs1, count2, nobs2, diff=0,
+                       alternative='two-sided'):
     r"""
-    Calculate E-test for the mean difference of two samples that follow a Poisson
-    distribution from descriptive statistics
+    Calculates the poisson mean test, the "e-test", for the mean difference of
+    two samples that follow a Poisson distribution from descriptive statistics.
+
+    This is a two-sided test for the null hypothesis that two independent
+    samples have identical average (expected) values.
+
+    Let :math:`X_{11},...,X_{1n_1}` and :math:`X_{21},...,X_{2n_2}` be
+    independent samples, from :math:`Poisson(\lambda_1)` and
+    :math:`Poisson(\lambda_2)` distributions. It is well known that :math:`X_1`
+    and :math:`X_2` are independent:
 
-    Let :math:`X_{11},...,X_{1n_1}` and :math:`X_{21},...,X_{2n_2}` be independent
-    samples respectively, from :math:`Poisson(\lambda_1)` and :math:`Poisson(\lambda_2)`
-    distributions. It is well known that
-    
     .. math:: X_1 = \sum_{i=1}^{n_1} X_{1i} \sim Poisson(n_1\lambda_1)
-    
-    independently of
-    
+
     .. math:: X_2 = \sum_{i=1}^{n_2} X_{2i} \sim Poisson(n_2\lambda_2)
-    
-    Let `count1` and `count1` be the observed values of :math:`X_1` and :math:`X_2`,
-    respectively. The problem of interest here is to test
-    
+
+    Let `count1` and `count1` be the observed values of :math:`X_1` and
+    :math:`X_2`, respectively. The problem of interest here is to test
+
     .. math::
        H_0: \lambda_1 = \lambda_2 + \mathtt{diff} \quad vs. \quad
        H_a: \lambda_1 \ne \lambda_2 + \mathtt{diff}
-    
-    for `two-sided` hypothesis, where :math:`\mathtt{diff} \ge 0` is a given
```

Comment:
we changed how the h0 and ha defined in this documentation. the recent one used two-sided as an example compared to the old one which used right sided.